src/dsp_tools/commands/project/create/project_create.py

```diff
@@ -117,32 +107,30 @@ def _update_basic_info_of_project(
 
     Args:
         project: the project to be updated (must exist on the DSP server)
-        shortcode: shortcode of the project (must be the same as in the existing project)
-        shortname: shortname of the project (must be the same as in the existing project)
-        longname: longname of the project
-        descriptions: descriptions of the project in different languages
-        keywords: keywords of the project
+        project_definition: class with project info
```

Comment:
to be very precise, you cannot hand over a class to a function, but rather an object. 
(Occurs several times)